server/src/main/java/org/apache/druid/guice/StorageNodeModule.java

```diff
@@ -52,6 +55,7 @@ public void configure(Binder binder)
   {
     JsonConfigProvider.bind(binder, "druid.server", DruidServerConfig.class);
     JsonConfigProvider.bind(binder, "druid.segmentCache", SegmentLoaderConfig.class);
+    JsonConfigProvider.bind(binder, "druid.segmentCache.locationSelector.strategy", StorageLocationSelectorStrategy.class);
```

Comment:
Should this be `druid.segmentCache.locationSelector` instead?

Reply:
Oh sorry, this is a bug. I remember I tested it in my cluster, maybe I missed something then. The code has been updated in the latest commit.